src/modules/customfields/helpers/field-encryption.service.ts

```
@@ -0,0 +1,236 @@
+			case FieldType.NUMERIC:
+			case FieldType.CURRENCY:
+			case FieldType.PERCENT:
+			case FieldType.RATING:
+				if (isNaN(Number(value))) {
+					throw new BadRequestException(`Value must be a valid number for field type ${fieldType}`);
+				}
+				break;
+
+			case FieldType.DATE:
+			case FieldType.DATETIME:
+				const date = new Date(value);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)